README.md

```diff
@@ -195,7 +195,8 @@ python setup.py install
 
 On macOS
 ```bash
-MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py install
+os=$(sw_vers -productVersion | cut -d '.' -f 1,2)
+MACOSX_DEPLOYMENT_TARGET="$os" CC=clang CXX=clang++ python setup.py install
```

Comment:
`MACOSX_DEPLOYMENT_TARGET` is the minimum target it should compile for, not exact version.

If we dont specify anything it compiles for 10.6.

We do 10.9 because until 10.8, the libc++ that was shipped didn't have C++11 support.